### PR TITLE
Added support for custom mpd port 

### PR DESCRIPTION
**Pull request by Blackhippy**

### CODE REVIEW

File: bum/song.py [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
 
 
-def init():
+def init(port=6600):
     """Initialize mpd."""
     client = mpd.MPDClient()
 
     try:
-        client.connect("localhost", 6600)
+        client.connect("localhost", f"{port}")
```
Comment (dylanaraps) [inline bum/song.py]: Just `port` can be used here. Or if that doesn't work: `str(port)`.

File: bum/__main__.py [hunk 2]
```diff
@@ -33,6 +33,10 @@ def get_args():
     arg.add_argument("--version", action="store_true",
                      help="Print \"bum\" version.")
 
+    arg.add_argument("--port",
+                     help="What is you MPD port.",
```
Comment (dylanaraps) [inline bum/__main__.py]: Change this to `Use a custom mpd port`.

File: README.md [hunk 3]
```diff
@@ -43,6 +43,7 @@ optional arguments:
   --cache_dir "/path/to/dir"
                         Where to store the downloaded cover art.
   --version             Print "bum" version.
+  --port                custom Mpd port
```
Comment (dylanaraps) [inline README.md]: Change this to `Use a custom mpd port`.

**Comment by dylanaraps** [comment]
Thanks for the PR, I left some comments.

File: bum/song.py (hunk 1 above)
Comment (Blackhippy) [reply bum/song.py]: Yes just <code> port </code> worked.

**Comment by Blackhippy** [comment]
Added Changes

**Comment by dylanaraps** [comment]
Thanks!!

**Pull request merged**